ci/lint.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.16.2
+    tag: 1.17.1
```

Comment:
I think the agreement for lint is to use `latest` here instead of a fixed version

Reply:
done, thanks